### PR TITLE
Set up initial configuration and webpage

### PR DESCRIPTION
**Pull request by Implausibility**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 2018,
+  },
+  plugins: [
+    'react',
+  ],
+  rules: {
+  },
```
Comment (eric1052) [inline .eslintrc.js:26]: Minor suggestion - perhaps you can get rid of this entirely if you have no rules to specify.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+require('dotenv').config();
+const path = require('path');
+const express = require('express');
+const { createProxyMiddleware } = require('http-proxy-middleware');
+
+const proxyOptions = {
+  target: 'localhost:9000',
+  changeOrigin: true,
+
```
Comment (eric1052) [inline index.js:9]: Unnecessary extra line

File: index.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+require('dotenv').config();
+const path = require('path');
+const express = require('express');
+const { createProxyMiddleware } = require('http-proxy-middleware');
+
+const proxyOptions = {
+  target: 'localhost:9000',
+  changeOrigin: true,
```
Comment (eric1052) [inline index.js:8]: I'm curious, what was your reason behind setting changeOrigin as true?

File: .eslintrc.js (hunk 1 above)
Comment (Implausibility) [reply .eslintrc.js:26]: Good point! I'll do that tomorrow.

File: index.js (hunk 3 above)
Comment (Implausibility) [reply index.js:8]: The documentation on npm says that that's the way to make it work when you're running on a local machine, as I recall.

This configuration is not finished. I'm not even using the proxy properly yet, just loading in the other pages raw. I just left the options there that I knew I needed later.

**Comment by Implausibility** [comment]
Thanks, @eric1052 !

**Pull request merged**